docs/markdown/Python/python/python-third-party-dependencies.md

```diff
@@ -533,6 +521,104 @@ the user:
 indexes.add = ["http://$USERNAME:$PASSWORD@my.custom.repo/index"]
 ```
 
+### Local requirements
+
+There are two ways to specify local requirements from the filesystem:
+
+* [PEP 440 direct references](https://www.python.org/dev/peps/pep-0440/#direct-references)
+
+```python 3rdparty/python
+python_requirement(
+    name="django",
+    # Use an absolute path to a .whl or sdist file.
+    requirements=["Django @ file:///Users/pantsbuild/prebuilt_wheels/django-3.1.1-py3-none-any.whl"],
+)
+
+# Reminder: we could also put this requirement string in requirements.txt and use the
```

Comment:
I expect this to be the more common thing, so why not just use that for demonstration?